ArmaForces.Boderator/ArmaForces.Boderator.BotService/Controllers/DebugController.cs

```diff
@@ -10,17 +12,26 @@ namespace ArmaForces.Boderator.BotService.Controllers
     public class DebugController : Controller
     {
         private readonly IMediator _mediator;
+        private readonly IDiscordService _discordService;
 
-        public DebugController(IMediator mediator)
+        public DebugController(IMediator mediator, IDiscordService discordService)
         {
             _mediator = mediator;
+            _discordService = discordService;
         }
 
-        [HttpGet]
+        [HttpGet("Test")]
         public async Task<ActionResult> Test(CancellationToken cToken)
         {
             string response = await _mediator.Send(new TestCommand(), cToken);
             return Ok(response);
         }
+
+        [HttpGet("GetDiscordStatus")]
+        public ActionResult GetDiscordStatus() => Ok(_discordService.GetDiscordClientStatus());
+
+        [HttpPut("SetBotStatus/{status}")]
+        public async Task<ActionResult> SetBotStatus(string status, ActivityType type = ActivityType.Playing) =>
+            await Task.Run(() => Ok(_discordService.SetBotStatus(status, type)) as ActionResult);
```

Comment:
This seems very, very counter intuitive. `_discordService.SetBotStatus(status, type)` doesn't return anything but is used as an object parameter for `Ok()`. And `Ok()` doesn't need casting to `ActionResult`. I would to it like this:
```suggestion
        public async Task<ActionResult> SetBotStatus(string status, ActivityType type = ActivityType.Playing)
        {
            await _discordService.SetBotStatus(status, type);
            return Ok();
        }
```